### ISSUE TITLE
CDC dtests: heavy load topology change tests

### ISSUE THREAD

**Issue by haaawk** [scylladb/scylla#5391]
1. Setup a cluster
2. Create a table with CDC enabled
3. Start load on the cluster using cassandra stress
4. Add/Remove node to/from a cluster
5. Stop cassandra-stress
6. Verify some invariants on CDC Desc and CDC Log entries.

**Comment by haaawk** [comment]
We probably should create two tests here. One for addition and one for deletion of the node.

**Comment by slivne** [comment]
@piodul ping - did we do such dtests ?

**Comment by piodul** [comment]
> @piodul ping - did we do such dtests ?

@slivne No, we didn't. I think we didn't get around to doing it because we thought that SCT would be better suited for such kind of test. I'm not sure, though - @haaawk?

**Comment by haaawk** [comment]
I agree SCT is the right tool.

**Comment by slivne** [comment]
@haaawk  / @kbr-  / @aleksbykov so we do have this covered by SCT CDC tests - no ?

can I close this

**Comment by aleksbykov** [comment]
@slivne 
We have short and long longevities jobs for cdc for 4h and for 400Gb-3days. During these jobs different nemesis (including topology changes) are executed. But there is no validation that data are correctly stored in cdc log tables.
In addition we have @kbr- tests with data validation in sct and topology changes by nemesis, but not sure they are included in regular run of sct longevities jobs
